operations/power_management/Power_Off_Compute_Cabinets.md

```diff
@@ -36,57 +36,107 @@ HPE Cray standard EIA racks typically include two redundant PDUs. Some PDU model
 
 1. (`ncn-m#`) Check the power status in liquid-cooled cabinets before shutdown.
 
-    This example shows liquid-cooled cabinets 1000 - 1003.
+    Either use `sat status` or `cray capmc` to check. The `State` should be `Off` for every Chassis.
 
-    ```bash
-    cray capmc get_xname_status create --xnames x[1000-1003]c[0-7] --format json
-    ```
+    1. (`ncn-m001#`) Check the power status for every liquid-cooled cabinet Chassis.
 
-1. (`ncn-m#`) Check the power status for nodes in the standard racks before shutdown.
+       ```bash
+       sat status --types Chassis
+       ```
 
-    This example shows nodes in cabinets 3001 - 3003.
+       Example output.
 
-    ```bash
-    cray capmc get_xname_status create --xnames x300[1-3]c0s[1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35]b[1-4]n0 --format json
-    ```
+       ```text
+       +---------+---------+-------+------+---------+------+----------+----------+
+       | xname   | Type    | State | Flag | Enabled | Arch | Class    | Net Type |
+       +---------+---------+-------+------+---------+------+----------+----------+
+       | x1020c0 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c1 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c2 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c3 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c4 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c5 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c6 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
+       | x1020c7 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
```

Comment:
Do you want to use an ellipsis or something to show truncated output here and elsewhere? E.g.:

```suggestion
       | x1020c7 | Chassis | Off   | OK   | True    | X86  | Mountain | Sling    |
       ...
       +---------+---------+-------+------+---------+------+----------+----------+
```

Reply:
fixed